janusgraph-core/src/main/java/org/janusgraph/graphdb/database/idassigner/VertexIDAssigner.java

```diff
@@ -143,6 +143,11 @@ public void assignID(InternalRelation relation) {
         assignID(relation, null);
     }
 
+    public void assignID(InternalRelation relation, long relationId){
+        assignID(relation, null);
+        relation.setId(relationId);
```

Comment:
This method looks redundant to me. Why assigning auto-generated id first and then overriding it with custom id?

Reply:
You are correct, can remove the assigning auto-generated id part, directly set the customized id. Already modified it.

Reply:
Thanks a lot for the review! @li-boxuan